docker/package/tezos_setup_wizard.py

```diff
@@ -50,6 +50,12 @@
     "no": "Start the services this time only",
 }
 
+history_modes = {
+    "rolling": "Store only minimal data, deleting everything before a rolling checkpoint",
+    "full": "Store chain data enough to reconstruct the complete chain state",
+    "archive": "Store everything, very storage-demanding",
```

Comment:
Small wording suggestion:
```suggestion
    "rolling": "Store a minimal rolling window of chain data, lightest option",
    "full": "Store enough chain data to reconstruct the complete chain state",
    "archive": "Store all the chain data, very storage-demanding",
```